kong.conf.default

```diff
@@ -29,6 +29,17 @@
                                  # Each Kong process must have a separate
                                  # working directory.
 
+#port_map =                      # When running Kong behind layer 4 port mapping
+                                 # (or port-forwarding), e.g. with
+                                 # `docker run -p 80:8000`, you can use this
```

Comment:
rather than providing commands here, it would help to provide examples like docker, L4 LBs in the cloud